content/api/relay-webhooks.apib

```diff
@@ -164,6 +175,18 @@ Create a relay webhook by providing a relay webhooks object as the POST request
                 "name": "Replies Webhook",
                 "target": "https://webhooks.customer.example/replies",
                 "auth_token": "5ebe2294ecd0e0f08eab7690d2a6ee69",
+                "auth_type": "oauth2",
+                "auth_request_details": {
+                    "url": "http://client.example.com/tokens",
+                    "body": {
+                        "client_id": "CLIENT123",
+                        "client_secret": "9sdfj791d2bsbf",
+                        "grant_type": "client_credentials"
```

Comment:
Instead of requiring `grant_type` on `POST`, how about making `grant_type=client_credentials` the default, making the field optional, and having it be effectively read-only? Since there's only one allowed value, which is also the default. Is that future proof enough?

Reply:
got it, i just ripped this off webhooks api.  so you're saying we are only supporting grant_type client_credentials to start?

Reply:
AIUI that's the only one that works for non-interactive auth between 2 (as opposed to 3) actors.